src/ccmain/tesseractclass.h

```diff
@@ -897,6 +897,9 @@ class TESS_API Tesseract : public Wordrec {
   BOOL_VAR_H(tessedit_create_txt);
   BOOL_VAR_H(tessedit_create_hocr);
   BOOL_VAR_H(tessedit_create_alto);
+  BOOL_VAR_H(tessedit_create_page);
```

Comment:
_page_ is already used in a different context for several parameters, so maybe `tessedit_create_page_xml` would be clearer.